### PR TITLE
Oidc

### PR DESCRIPTION
**Pull request by TheOneRing**

### CODE REVIEW

File: src/libsync/creds/oauth.h [hunk 1]
```diff
@@ -59,7 +59,7 @@ class OWNCLOUDSYNC_EXPORT OAuth : public QObject
     OAuth(Account *account, QObject *parent);
     ~OAuth();
     void startAuthentification();
-    void refreshAuthentification(const QString &refreshToken);
+    void refreshAuthention(const QString &refreshToken);
```
Comment (ogoffart) [inline src/libsync/creds/oauth.h]: I did not know Authention is a word.  What's the reason behind this  change?

Comment (ogoffart) [reply src/libsync/creds/oauth.h]: Maybe it should be "Authentication" ?

File: src/libsync/creds/oauth.h [hunk 2]
```diff
@@ -59,7 +59,7 @@ class OWNCLOUDSYNC_EXPORT OAuth : public QObject
     OAuth(Account *account, QObject *parent);
     ~OAuth();
     void startAuthentification();
```
Comment (ogoffart) [inline src/libsync/creds/oauth.h]: This should also be renamed.

File: src/libsync/creds/oauth.h (hunk 1 above)
Comment (TheOneRing) [reply src/libsync/creds/oauth.h]: thx 🤦‍♀

**Pull request merged**